### PR TITLE
[internal review] Update test scripts for issue 3829

### PR DESCRIPTION
**Pull request by VjKlepikov**
ATF Test Scripts to check [FORDTCN-12279](https://luxproject.luxoft.com/jira/browse/FORDTCN-12279)

This PR is **[ready]** for review.

### Summary
Update test scripts for issue 3829

### ATF version
develop

### Changelog
Update test scripts

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/7_1/885_ShowConstantTBT.lua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---------------------------------------------------------------------------------------------------
 -- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/885
 ---------------------------------------------------------------------------------------------------
--- Description: Check that SDL stops processing softButton events once a response is received for the RPC.
+-- Description: Check that SDL does not stop processing softButton events once a response is received for the RPC.
```
Comment (dboltovskyi) [inline test_scripts/Defects/7_1/885_ShowConstantTBT.lua:4]: I would add a link to `3829` issue along with the previous `885`

**Review by dboltovskyi — APPROVED** [review]

File: test_scripts/Defects/7_1/885_ShowConstantTBT.lua (hunk 1 above)
Comment (VjKlepikov) [reply test_scripts/Defects/7_1/885_ShowConstantTBT.lua:5]: @dboltovskyi Done in #150

**Review by VNikiforov-dev — APPROVED** [review]

**Comment by VjKlepikov** [comment]
@dboltovskyi please note that scripts have been updated

**Review by dboltovskyi — APPROVED** [review]

**Comment by VjKlepikov** [comment]
move to smartDeviceLink

**Pull request closed without merging**